system/system_monitor/launch/system_monitor.launch.py

```diff
@@ -104,6 +114,7 @@ def launch_setup(context, *args, **kwargs):
         package="rclcpp_components",
         executable="component_container_mt",
         composable_node_descriptions=[
+            voltage_monitor,
```

Comment:
Because it dose not work if it's added another line.
I didn't know what caused the problem.
Here is the log when I add it in the second line.
```
        executable="component_container_mt",
        composable_node_descriptions=[
            cpu_monitor,
            voltage_monitor,
            hdd_monitor,
```
```[INFO] [launch]: All log files can be found below /home/autoware/.ros/log/2022-09-14-23-56-31-032719-autoware-ROScube-I-ET-1474796
[INFO] [launch]: Default logging verbosity is set to INFO
[INFO] [launch_ros.actions.load_composable_nodes]: Loaded node '/system/system_monitor/cpu_monitor' in container '/system/system_monitor/system_monitor/system_monitor_container'
[ERROR] [launch_ros.actions.load_composable_nodes]: Failed to load node 'voltage_monitor' of type 'VoltageMonitor' in container '/system/system_monitor/system_monitor/system_monitor_container': Failed to find class with the requested plugin name.
[INFO] [launch_ros.actions.load_composable_nodes]: Loaded node '/control/external_cmd_converter' in container '/control/control_container'
[INFO] [launch_ros.actions.load_composable_nodes]: Loaded node '/awapi/route_relay' in container '/awapi/awapi_relay_container'
[INFO] [launch_ros.actions.load_composable_nodes]: Loaded node '/system/system_monitor/hdd_monitor' in container '/system/system_monitor/system_monitor/system_monitor_container'
```

Reply:
I made some changes and it works.
- system_monitor.launch.py 
  ![image](https://user-images.githubusercontent.com/57388357/190262276-2898d790-17af-44ae-8d34-f4375e8d948d.png)
- system_monitor.launch.xml
  ![image](https://user-images.githubusercontent.com/57388357/190262635-8f24e699-a437-48bb-b2ec-a06479386707.png)

- CMakeLists.txt
  ![image](https://user-images.githubusercontent.com/57388357/190263218-e8155522-e869-4054-80a3-cbe4d55ed910.png)

  ![image](https://user-images.githubusercontent.com/57388357/190263353-6c92f766-d44a-40dc-8099-7e5e97bc1c83.png)

- `ros2 launch system_launch system.launch.xml sensor_model:=aip_x1`
  ![image](https://user-images.githubusercontent.com/57388357/190264131-4a848f2c-430d-4159-871c-6b6a1e5d3154.png)

- `ros2 run rqt_runtime_monitor rqt_runtime_monitor`
  ![image](https://user-images.githubusercontent.com/57388357/190264209-f0dff747-8b02-4f0c-a0a9-26ef8f38a6d2.png)

Reply:
thank you.
moved voltage_monitor from the top to the bottom.